desc/objectives/_free_boundary.py

```diff
@@ -364,6 +377,13 @@ class BoundaryError(_Objective):
     loop : bool
         If True, evaluate integral using loops, as opposed to vmap. Slower, but uses
         less memory.
+    sz : int
```

Comment:
can this go next to `st`? or do we want it last bc it should be set by defauly or something?

Reply:
That would be better to change after #1458 . Don't want people to mix q and sz